README.rst

```diff
@@ -694,6 +694,19 @@ Choose from the list of available rules:
 
   *Risky rule: risky if any of the functions ``intval``, ``floatval``, ``doubleval``, ``strval`` or ``boolval`` are overridden.*
 
+* **native_constant_invocation** [@Symfony:risky]
+
+  Add leading ``\`` before constant invocation of internal constant to speed
+  up resolving. Constant name match is case-sensitive.
+
+  *Risky rule: risky when any of the constants are namespaced or overridden.*
+
+  Configuration options:
+
+  - ``exclude`` (``array``): list of constants to ignore; defaults to ``[]``
+  - ``include`` (``array``): list of constants to fix. Default is to fix all
```

Comment:
I must admit it confuses me.
> Default is to fix all constants returned by ``get_defined_constants``; defaults to ``[]``

If I wouldn't know what we actually do in the fixer, I wouldn't know what is default.

Reply:
IMO, this proves it's one option to cover two scenarios.
I would split it into `fix_built_in: boolean` to decide whether they should be fixed or not, and on top of that `include: array` to add sth on top of that.
then, together with https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3127/files#r150393437 , default config would be `fix_built_in: true, exclude: ['true', 'false', 'null']`